src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -122,7 +166,7 @@ trait RegularSync {
           val (newBlocks, errorOpt) = processBlocks(blocks, blockParentTd)
 
           if(newBlocks.nonEmpty){
-            context.self ! BroadcastBlocks(newBlocks)
+            handshakedPeers.keys.foreach(actor => actor ! BroadcastBlocks(newBlocks))
```

Comment:
Does that mean we'll send the same block to the same peer we received it from? Should it be "smarter"?